build_test.go

```diff
@@ -194,8 +185,9 @@ func testBuild(t *testing.T, context spec.G, it spec.S) {
 
 	context("when the conda layer is required at build and launch", func() {
 		it.Before(func() {
-			entryResolver.MergeLayerTypesCall.Returns.Launch = true
-			entryResolver.MergeLayerTypesCall.Returns.Build = true
+			buildContext.Plan.Entries[0].Metadata = make(map[string]interface{})
```

Comment:
This can be inlined - see https://github.com/paketo-buildpacks/pip/pull/256#discussion_r868058514